test/controllers/user_controller_test.exs

```diff
@@ -126,6 +126,7 @@ defmodule CodeCorps.UserControllerTest do
       assert user.biography == "Just a test user"
     end
 
+    @tag :s3
```

Comment:
Do we see there being a distinction between interacting with third party services / general / other secret env variables? I'm just wondering how granular we want to make it?

I guess we could always make it generically `requires_env` now then get more granular in the future if we ever need to.